### DISCUSSION TITLE
Support for 'beta' releases?

### DISCUSSION THREAD

**Discussion by theothernt** [Sharkaboi/AppUpdateChecker#3] [Q&A]
Does **AppUpdateChecker** have the ability to deal with stable vs beta releases?

For example, [SmartTubeNext](https://github.com/yuliskov/SmartTubeNext) has its own auto-update feature with a 'stable' and 'beta' track.

**Comment by Sharkaboi** [comment]
Hey

Sorry for responding so late, seems like I had notifications/email off for discussions.

What is the source you want to get latest versions from?

With the latest version of `AppUpdateChecker`, depending on the source for the versions, we can have either : 
- one instance for each track pointing to different tracks
- swap out URLs based on track in app code (for JSON/XML endpoint)
- Write a custom source that gets latest version of each track. (Currently github and fdroid sources only look at latest release)
